src/app/common/components/CertificateUpload.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { render } from 'react-dom';
+import {
+  Button,
+  InputGroup,
+  TextInput
+} from '@patternfly/react-core';
+
+interface ICertificateUploadProps {
+  isDisabled: boolean;
+  onBlur: any;
+  onChange: any;
+  onInput: any;
+  name: string;
+}
+
+interface ICertificateUploadState {
+  display: string;
+}
+
+class CertificateUpload extends React.Component<ICertificateUploadProps, ICertificateUploadState> {
```

Comment:
We are using functional components across the app. Would be great to convert this to a function component.

Reply:
```const CertificateUpload: React.FunctionComponent<ICertificateUploadProps> = ({
  isDisabled, 
  onBlur, 
  onInput, 
  setFieldValue, 
  name
})=> {
```